### PR TITLE
Adding sources

### PR DESCRIPTION
**Pull request by dirkseidensticker**
**Don't merge!**

Initial parser for a data set that was published by Courtney Mustaphi and Marchant in 2016. The data itself are located on the Dataverse-Server of Harvard University and can't be accessed directly. Therefor my parser downloads the CSV file first to read it in later. I'm not entirely sure if this is the best approach.

@nevrome : Please review and comment on initial parser with focus on `tidyr`-approach

Once I'm more confident with replicating your ( @nevrome  ) approach I'll try to add more parsers for:

- [x] AustArch: A Database of 14C and Luminescence Ages from Archaeological Sites in Australia: http://archaeologydataservice.ac.uk/archives/view/austarch_na_2014/ & http://archaeologydataservice.ac.uk/catalogue/adsdata/arch-1661-1/dissemination/csv/Austarch_1-3_and_IDASQ_28Nov13-1.csv & http://archaeologydataservice.ac.uk/archiveDS/archiveDownload?t=arch-1661-1/dissemination/csv/Austarch_1-3_and_IDASQ_28Nov13-1.csv
- [ ] Radiocarbon Palaeolithic Europe Database v21: http://ees.kuleuven.be/geography/projects/14c-palaeolithic/
- [ ] Flohr 2015: http://www.sciencedirect.com/science/article/pii/S0277379115300317
- [ ] Wang 2014: http://www.sciencedirect.com/science/article/pii/S0277379114001966

### CODE REVIEW

File: R/eastafrica.R [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+#' \dontrun{
+#'   KITEeastAfrica <- get_KITEeastAfrica()
+#' }
+#'
+#' @export
+get_KITEeastAfrica <- function() {
+
+  Sys.setenv("DATAVERSE_SERVER" = "dataverse.harvard.edu")
+
+  writeBin(get_file("CARD Upload Template - KITE East Africa v2.1.csv", "doi:10.7910/DVN/NJLNRJ"), "data-raw/KITE_EastAfrica.csv")
```
Comment (nevrome) [inline R/eastafrica.R]: Please work with a temporary file like I did [here](https://github.com/ISAAKiel/c14bazAAR/blob/4e34cf0a26057916f8cf763ea399c408b2dc71e7/R/14sea.R#L14-L25) and add the internet connection check. 

c14bazAAR should not contain any data by itself. This is not a data- but an API package.

File: R/eastafrica.R [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        'MA Sigma' = '_',
+        'Delta 13 C Source' = '_',
+        'Significance' = '_',
+        'Site Name' = '_',
+        'Stratigraphic Component' = '_',
+        'Context' = '_',
+        'Associated Taxa' = '_',
+        'Additional Information' = '_',
+        'Comments' = '_'
+        )) %>%
```
Comment (nevrome) [inline R/eastafrica.R]: Please add [col_type determination](https://github.com/ISAAKiel/c14bazAAR/blob/4e34cf0a26057916f8cf763ea399c408b2dc71e7/R/adrac.R#L24-L36).

File: R/eastafrica.R [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    dplyr::rename(
+      labnr = .data[["Lab Number"]],
+      c14age = .data[["Normalized Age"]],
+      c14std = .data[["NA Sigma"]],
+      c13val = .data[["Delta 13C (per mil)"]],
+      material = .data[["Material Dated"]],
+      site = .data[["Site Identifier"]],
+      lat = .data[["Latitude"]],
+      lon = .data[["Longitude"]],
+      shortref = .data[["References"]]
```
Comment (nevrome) [inline R/eastafrica.R]: The additional information can also be part of the dataset. Just rename it accordingly (e.g. "Stratigraphic Component" -> stratigraphic_component or "Additional Information" ->comment (?)). 

We need a definite list of the variable names... An object of class c14_date_list can store any number of additional variables and is valid as long as the basic ones are present. And even here we have to be flexible.

**Comment by nevrome** [comment]
Thanks for this first parser, Dirk. I realize now, that some essential questions are still open. Let's talk about this in person.

File: R/eastafrica.R (hunk 3 above)
Comment (dirkseidensticker) [reply R/eastafrica.R]: Concerning defined list of variable names, what do think about a simple markdown document that holds all variables currently pulled from all the different parsers? People could reference those and add new field if necessary? See https://github.com/dirkseidensticker/aDRAC/blob/master/data/aDRAC.md for what I've in mind.

**Comment by dirkseidensticker** [comment]
The AustArch Parser has two issues:

- [ ] As the original CSV has a trailing comma and thus the dataset a empty column; I didn't found a readr::read_csv-method that can handle that
- [ ] the `AGE`/`c14age` contains some non-integer values that got droped; I'm not sure if that's what we want in such cases?

**Comment by dirkseidensticker** [comment]
@nevrome just realized you build a parser for AustArch as well. Which version should we keep. How do you think about merging this branch into master? The missing datasets are all behind login or pay-wall.

**Pull request merged**